packages/blocks/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -0,0 +1,125 @@
+import React, { useState } from 'react';
+import cx from 'classnames';
+import { PlusIcon, SearchIcon } from '@mergestat/icons';
+import { Tag } from '../Filter/Tag';
+import { Checkbox } from '../Form';
+
+type MultiSelectProps = {
+  setStateToProps: {
+    title: string;
+    checked: boolean;
+  }[];
+  getState?: ([]) => void;
+};
+
+export const MultiSelect: React.FC<
+  MultiSelectProps &
+    React.DetailedHTMLProps<
+      React.DetailsHTMLAttributes<HTMLElement>,
+      HTMLElement
+    >
+> = ({ setStateToProps, getState }) => {
+  const [state, setState] = useState(setStateToProps);
+  const [value, setValue] = useState('');
+  const [isActive, setIsActive] = useState(false);
+
+  window.addEventListener('click', (e) => {
+    const target = e.target as Element;
+    if (!['input', 'span'].includes(target.localName)) {
+      setIsActive(false);
+    }
+  });
+
+  window.addEventListener('keypress', (e) => {
+    if (e.key === 'Enter' && value.length > 0) {
+      setState([...state, { title: value, checked: true }]);
+
+      if (getState) {
+        getState([...state]);
+      }
+      setValue('');
+    }
+  });
```

Comment:
I think we should make sure to remove these event listeners as well, when the component is unmounted, `removeEventListener` - see [here](https://www.pluralsight.com/guides/how-to-cleanup-event-listeners-react)